integration-cli/daemon/daemon.go

```diff
@@ -222,7 +222,7 @@ func (d *Daemon) StartWithLogFile(out *os.File, providedArgs ...string) error {
 		return errors.Wrapf(err, "[%s] could not find docker binary in $PATH", d.id)
 	}
 	args := append(d.GlobalFlags,
-		"--containerd", "/var/run/docker/libcontainerd/docker-containerd.sock",
+		"--containerd", "/var/run/docker/containerd/docker-containerd.sock",
```

Comment:
We should check upgrades, especially with live-restore.
Suspect live-restore will not work, so users will need a way to to know how to upgrade properly.
@thaJeztah